crates/pop-parachains/src/build.rs

```diff
@@ -72,15 +72,19 @@ fn binary_path(target_path: &Path, node_path: &Path) -> Result<PathBuf, Error> {
 /// * `binary_path` - The path to the node binary executable that contains the `build-spec` command.
 /// * `plain_chain_spec` - Location of the plain_parachain_spec file to be generated.
 /// * `para_id` - The parachain ID to be replaced in the specification.
+/// * `default_bootnode` - Whether to include localhost as a bootnode.
 pub fn generate_plain_chain_spec(
 	binary_path: &Path,
 	plain_chain_spec: &Path,
 	para_id: u32,
+	default_bootnode: bool,
 ) -> Result<(), Error> {
 	check_command_exists(&binary_path, "build-spec")?;
-	cmd(binary_path, vec!["build-spec", "--disable-default-bootnode"])
-		.stdout_path(plain_chain_spec)
-		.run()?;
+	let mut args = vec!["build-spec"];
+	if default_bootnode {
+		args.push("--disable-default-bootnode");
+	}
+	cmd(binary_path, args).stdout_path(plain_chain_spec).run()?;
```

Comment:
Check elsewhere for usages on redirecting std_err to avoid the process output messing up pop's output.